membership/build.gradle

```diff
@@ -16,23 +16,9 @@ android {
 dependencies {
     kapt libs.dagger_compiler
     kapt libs.dagger_android_compiler
-    kapt libs.glide_compiler
 
-    implementation libs.arch_navigation
-    implementation libs.arch_navigation_ui
-
-    api libs.design_support
-    implementation libs.glide
-    implementation project(':base')
-    implementation project(':ui')
-    implementation project(':viewmodel')
-    implementation project(':navigation')
-    implementation project(':search')
+    api project(':base')
```

Comment:
It really does feel nice to see all the unnecessary direct dependencies removed.